src/components/shared/transactionsTable/index.js

```diff
@@ -149,7 +150,7 @@ class TransactionsTable extends React.Component {
                         tooltipClassName={styles.tooltip}
                         content={<LiskAmount val={transaction.fee} token={tokenMap.LSK.key} />}
```

Comment:
```suggestion
                         content={<LiskAmount val={transaction.fee} token={tokenMap.LSK.key} />}
                         size="s"
```
I was adding size (to align tooltip padding) in https://github.com/LiskHQ/lisk-hub/commit/d0077e97bc165dda1feb37e2de3e12dd2fce9f4b#diff-2e0c48c8b82d2186028ea2f9a9286d29R79 but looks like it got lost in some merge conflict
https://github.com/LiskHQ/lisk-hub/blob/4948dc3601820ad487007bbdc9ea51366a442ec6/src/components/screens/monitor/blocks/blocks.js#L96

🎨 now it looks a bit weird
![Screenshot 2019-10-18 at 12 31 14](https://user-images.githubusercontent.com/1254342/67087844-09089a00-f1a4-11e9-8366-34b91b7bc146.png)

Reply:
After discussing it with Yashar, I have increased the size of the fee tooltip and make it show on the left, that way the end of the table doesn't overlap with the tooltip and it also aligns with the status behaviour one. I have added this as a different size to the `Tooltip` component. Let me know if this looks ok to you.